doc/md/aggregate.md

```diff
@@ -84,4 +84,57 @@ func Do(ctx context.Context, client *ent.Client) {
 		}).
 		Scan(ctx, &users)
 }
+```
+
+## Having + Group By
+
+This example requires custom SQL modifiers. See how to enable them [here](features.md#custom-sql-modifiers)
+
+In this example we're going to query the oldest users for each role.
+
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+
+	"entgo.io/ent/dialect/sql"
+	"<project>/ent"
+	"<project>/ent/user"
+)
+
+func Do(ctx context.Context, client *ent.Client) {
+	var users []struct {
+		Id    	Int
+		Age     Int
+		Role    string
+	}
+
+	client.User.Query().Modify(
+		func(s *sql.Selector) {
+			s.GroupBy(
+				User.Role,
+			)
+			s.Having(
+				sql.EQ(
+					user.FieldAge,
+					sql.Raw(sql.Max(user.FieldAge)),
+				),
+			)
+		}
+	).ScanX(ctx, &users)
+}
+
+```
+
+**Note:** The `sql.Raw` is crucial to have. It  tells the predicate that `sql.Max` or whatever wrapped function is not an arguement.
```

Comment:
you have double space after 'it'.
No need to "whatever " I think it's clear enough.

Reply:
fixed